### ISSUE TITLE
Organize `/list`

### ISSUE THREAD

**Issue by DUFCLiam** [TotalFreedom/TotalFreedomMod#1577]
Make `/list` like this:

`
[SrA] Senior1 [SrA] Senior2 [TCA] Telnet1 [TCA] Telnet2 [SA] Super1 [SA] Super2 [OP] OP1 [OP] OP2
`
Rather than mixed.

**Comment by Adaptified** [comment]
Who cares? It's color coded so it's easy to spot out.

**Comment by DUFCLiam** [comment]
I care, People with OCD (Do not confuse with OCPD)

**Comment by Adaptified** [comment]
@DUFCLiam OCD is not a condition that really affects someone, no need to fix things that aren't broken.

**Comment by DUFCLiam** [comment]
@Adaptified, This is an enhancement... Not a bug... We want to make things better.

**Comment by Wild1145** [comment]
I agree, I think they should be grouped, just like how DC does it, and just how quite a lot of servers do it nower days.

**Comment by OxVeganMasterxO** [comment]
It doesn't really, I like to keep it personal but I have OCD and I find it easy OCD is sort of a different thing (e.g overreactions)

**Comment by OxVeganMasterxO** [comment]
Although It would look nice,tidy and neater
